DataFormats/METReco/src/classes_def.xml

```diff
@@ -164,14 +164,18 @@
   </class>
   <class name="edm::Wrapper<reco::EcalHaloData>"/>
 
-  <class name="reco::HcalHaloData" ClassVersion="10">
+  <class name="reco::HcalHaloData" ClassVersion="11">
    <version ClassVersion="10" checksum="3928363167"/>
+   <version ClassVersion="11" checksum="3305429659"/>
   </class>
   <class name="edm::Wrapper<reco::HcalHaloData>"/>
 
+  <class name="std::vector<HaloTowerStrip>"/>
+  <class name="edm::Wrapper<std::vector<HaloTowerStrip> >"/>
+
   <class name="reco::CSCHaloData" ClassVersion="11">
```

Comment:
please keep the lines of CSCHaloData  unmodified (it's not a part of this PR change)